sparksteps/steps.py

```diff
@@ -160,10 +160,14 @@ def upload_steps(s3_resource, bucket, path):
         copy_step = CopyStep(bucket, basename + '.zip')
         zip_to_s3(s3_resource, path, bucket, key=copy_step.key)
         steps.extend([copy_step, UnzipStep(path)])
-    else:
+    elif os.path.isfile(path):
         copy_step = CopyStep(bucket, basename)
         s3_resource.meta.client.upload_file(path, bucket, copy_step.key)
         steps.append(copy_step)
+    else:
+        raise ValueError(
```

Comment:
FileNotFound seems like the appropriate exception here, no?

https://docs.python.org/3/library/exceptions.html#FileNotFoundError

Reply:
+1

Reply:
I think that's more appropriate, my thinking for ValueError was that it's a faulty user input but I can change it.